scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -254,6 +258,42 @@ function doImportEclipse() {
   "${ECLIPSE_HOME}/eclipse" -nosplash -vm "${ECLIPSE_JAVA_HOME}/bin/java" -application org.eclipse.ant.core.antRunner -buildfile "${DEVON_IDE_HOME}/scripts/lib_script/import.xml" -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${ECLIPSE_JAVA_HOME}/bin/java" -vmargs -DdevonImportPath="${importPath}" -DdevonImportWorkingSet="${importWorkingSets}" ${ECLIPSE_VMARGS} 
 }
 
+# $1: URL
+# $2: name
+# $3: optional dest
```

Comment:
could we also support the existing plugins configuration?
So if only one argument is provided get the URL from according plugin properties file and the name from `$1`.
So if `checkstyle` is provided then it reads the update site from here:
https://github.com/devonfw/ide-settings/blob/master/eclipse/plugins/checkstyle.properties#L1

There is already code to add plugins handling this stuff that could be reused.

Reply:
Maybe we could even add a `plugin_mirror_url` property overriding `plugin_url` for plugin installations so I can always just use one param instead of two?

Reply:
Should I extend the  `add-plugin` function so that if `plugin_mirror_url` is set, this URL is used instead of `plugin_url`?

Reply:
I have now added two parameters for the mirror function. The first ist the id of the plugin (e.g. checkstyle) and the second is an optional URL. If there is only one parameter, the  `plugin_url` will be extracted automatically from the related plugin property file. If the optional URL is set as second parameter, the function will mirror this URL instead. 

I choose this structure because in my opinion the first parameter (plugin id) is needed in any case, because the download folder is named by this. With only an URL as paramter I would have to implement something for the name. Moreover, the user only needs to pass a single world (e.g. checkstyle) as a parameter instead of the path of the related property file, if we were to take the property file as a parameter.